code/game/antagonist/antagonist_add.dm

```diff
@@ -32,8 +33,8 @@
 		return 0
 	current_antagonists |= player
 
-	if(faction_verb && player.current)
-		add_verb(player.current.client, faction_verb)
+	if(length(faction_verbs) && player.current)
```

Comment:
```suggestion
	if(LAZYLEN(faction_verbs) && player.current)
```

`faction_verbs` appear to be a lazy list (you're not initializing it above where it's declared) and should therefore be accessed with the appropriate macros